index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap"
+			rel="stylesheet" />
+		<link rel="stylesheet" href="style.css" />
+		<title>Lincoln's Portfolio</title>
+	</head>
+	<body>
+		<header class="header">
+			<nav class="nav">
+				<a href="#">
+					<img class="menu-icon" src="/images/Icon - Menu.png" alt="" />
+				</a>
+			</nav>
+		</header>
+		<main class="intro">
+			<section class="intro-container">
+				<h1 class="intro-heading">
+					<span
+						>Hey there. <br />
+						I'm Lincoln.</span
+					>
+					<br />
+					I'm a software developer
+				</h1>
+				<div class="intro-description">
+					<p>
+						I can help you build a product , feature or website Look through
+						some of my work and experience! If you like what you see and have a
+						project you need coded, don't hesitate to contact me.
+					</p>
+				</div>
+				<section class="social">
+					<ul class="social-links">
+						<li class="social-link">
+							<a href="https://twitter.com/"
+								><img src="/images/twitter.svg" alt="" />
+							</a>
+						</li>
+						<li class="social-link">
+							<a href="https://www.linkedin.com/in/"
+								><img src="/images/Linkedin icon.svg" alt="" />
+							</a>
+						</li>
+						<li class="social-link">
+							<a href="https://medium.com/"
+								><img src="/images/medium.svg" alt="" />
+							</a>
+						</li>
+						<li class="social-link">
+							<a href="https://github.com/"
+								><img src="/images/github.svg" alt="" />
+							</a>
+						</li>
+						<li class="social-link">
+							<a href="https://angel.com/"
+								><img src="/images/angellist.svg" alt="" />
```

Comment:
**OPTIONAL** - I currently can not see any image on your website because you are navigating into the `images` dir. wrongly, please use a `./images/..../` when navigating into your image dir.. Please fix this 🙏 

Your project:
![tjsthseth](https://user-images.githubusercontent.com/73607512/227202650-6576d356-8ccd-4349-ae33-1d2ec0c15505.PNG)